### ISSUE TITLE
Phase bar improvements

### ISSUE THREAD

**Issue by LevelX2** [magefree/mage#172]
1) Add a phase bar per user area. So each user has his own phase bar that is only used during its turn. It would be much more intuitive to know whose turn it is.

2) Like the player icon, the active phase bar gets a colored frame that indicates whose turn it is.
3) Show the defined phases to stop in the phase bar by setting a small dot/mark within the phase icon.

4) Allow to toggle the settings for stopping at a phase by pushing the phase icon of the own or opponent phase bar.

**Comment by aivopaas** [comment]
I'd like to add that indication of who's turn it is should be much more prominent. Something like in mtgo that uses background color to indicate turns. I have played only a few games with XMage and messed up too many times skipping to my turn while it is my turn and I haven't played a land. The problem is I thought it was still opponents turn because when I started pressing the skip button, it was still their precombat main phase. Obviously that happens mostly in the first few turns when there's nothing but lands to play.

And similarly, when I have priority, it should be clearly visually distinctive so that I can quickly click Done to skip to the point where I actually want to take action. Currently I have to keep staring at the tiny text and parse it manually or wait for the Done to pop up. Having a 27 inch screen only adds to the burn.
